kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/asset/Asset.java

```diff
@@ -69,7 +67,7 @@
      * @return the list of asset records which comprises the currently read
      *         value in case of success or the reason of failure
      */
-    public List<AssetRecord> read(List<Long> channelIds) throws KuraException;
+    public List<AssetRecord> read(Set<String> channelNames) throws KuraException;
```

Comment:
Yes, but this shouldn't be a problem because the channel name is case-sensitive, right? Two strings with different cases refer to two different channels.